packages/devtools_app/lib/src/charts/flame_chart_canvas.dart

```diff
@@ -243,11 +243,11 @@ abstract class FlameChartCanvas<T> extends FlameChart {
   // canvas settings.
   void _paintCallback(CanvasRenderingContext2D canvas, Rect rect) {
     final int startRow = math.max(rowIndexForY(rect.top), 0);
-    final int endRow = math.min(
-      rowIndexForY(rect.bottom) + 1,
+    final int endRowInclusive = math.min(
```

Comment:
nit: it would be more standard to write
```dart
final int endRow = math.min(
 rowIndexForY(rect.bottom),
 rows.length,
);

for (int i = startRow; i < endRow; i++) {
```

Reply:
done.